DisguiseUnityRenderStream/Editor/DisguiseRenderStreamSettingsProvider.cs

```diff
@@ -0,0 +1,37 @@
+using UnityEditor;
+using UnityEngine.UIElements;
+
+namespace Disguise.RenderStream
+{
+    static class DisguiseRenderStreamSettingsProvider
+    {
+        static readonly string k_SettingsPath = "Project/DisguiseRenderStream";
+
+        class Contents
+        {
+            public const string SettingsName = "Disguise RenderStream";
+        }
+
+        [SettingsProvider]
+        static SettingsProvider CreateSettingsProvider() =>
+            new (k_SettingsPath, SettingsScope.Project)
+            {
+                label = Contents.SettingsName,
```

Comment:
Food for thoughts: I noticed that when the Cluster Display package looks like is present settings looks like this:
![image](https://user-images.githubusercontent.com/103442493/229773903-b78cc90b-5caf-40e8-9c9b-e6ed82641896.png)

I wonder, is it what we want?  Because there are a lot of settings in there that are not for Disguise RenderStream and could potentially confuse the user (and cause trouble if wrongly configured).  In fact, when Cluster Display is working with Disguise I'd be tempted to have the following settings:
Disguise RenderStream:

- Scene Control
- Unity Debug Window Presenter
- Cluster Synchronization (as a "sub section", like Cluster Render and Mission Control is to Cluster Display)
  - Multicast Address
  - Port
  - Handhsake Timeout
  - Communication Timeout
  - Adapter Name

But I guess this cannot be achieved only in this package, we also need a way for the ClusterDisplay package to stop providing its own settings when "requested" (or maybe we simply want to hardcode checking for the DisguiseRenderStream package)

Reply:
That's interesting. I rather like the idea of making Cluster Display (core) a "hidden" dependency of other packages (e.g. Graphics, RenderStream), kind of similar to Core SRP. Let's take a look at how Core SRP settings are exposed for inspiration.